### PR TITLE
Added a new Field in FormBuilder to support Radio Groups

### PR DESCRIPTION
**Pull request by wshamim**

### CODE REVIEW

File: src/components/TreeActionPanel/FormBuilder.js
Comment (michaeljharrison) [inline src/components/TreeActionPanel/FormBuilder.js]: Should we be using === here for good practice?

File: src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json
Comment (michaeljharrison) [inline src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json]: Change to "Use $or"

File: src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json
Comment (michaeljharrison) [inline src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json]: Change to "Use $and"

File: src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json
Comment (michaeljharrison) [inline src/components/AggregateViews/AggregateBlocks/BlockDefinitions/Match.ddd.json]: Change to "Default"

**Review by michaeljharrison — CHANGES_REQUESTED** [review]
Functionally looks good, few minor changes to make before we merge it, I've added comments to the relevant sections but here is an overall:

- Ability to add Tooltips for individual radio elements.
- Swap SimpleQuery Or and And fields to this new radio button component.
- Get styling approved / created by Kalid, if he's happy with it I'm happy with it.

**Comment by wshamim** [comment]
I have incorporated the changes which you have requested. Have a look now and let me know if anything else is required.

**Review by michaeljharrison — APPROVED** [review]
Looks good.

**Pull request merged**
